lib/models/subscription_option_wrapper.dart

```diff
@@ -28,20 +28,21 @@ class SubscriptionOption with _$SubscriptionOption {
     /// Pricing phases defining a user's payment plan for the product over time.
     @JsonKey(name: 'pricingPhases') List<PricingPhase> pricingPhases,
 
-    /// Tags defined on the base plan or offer. Empty for Amazon.
+    /// Tags defined on the base plan or offer.
     @JsonKey(name: 'tags') List<String> tags,
 
     /// True if this SubscriptionOption represents a Google subscription base plan (rather than an offer).
-    /// Not applicable for Amazon subscriptions.
     @JsonKey(name: 'isBasePlan') bool isBasePlan,
 
     /// The subscription period of fullPricePhase (after free and intro trials).
     @JsonKey(name: 'billingPeriod') Period? billingPeriod,
 
+    /// True if the subscription is pre-paid.
+    @JsonKey(name: 'isPrepaid') bool isPrepaid,
```

Comment:
I wonder if this property should live in the `StoreProduct` model... As in, in our abstraction, StoreProduct corresponds to a base plan, which is what will be linked to be `prepaid` or not... Thoughts?

Reply:
@tonidero This is probably a good point! I'll start the PR changing up from native Android to start that 😛